src/BridgedRoom.ts

```diff
@@ -429,6 +460,10 @@ export class BridgedRoom {
         if (message.user_id === this.team!.user_id) {
```

Comment:
I think we want to drop this clause, because we need to handle events that the bot sent to add them to the db.

Reply:
Can we leave this for your PR?

Reply:
yep